sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -594,4 +594,12 @@ internal class NativeMapImpl(val map: MapInterface) :
   override fun unsubscribe(observer: Observer) {
     map.unsubscribe(observer)
   }
+
+  override fun setRenderWorldCopies(renderWorldCopies: Boolean) {
+    map.renderWorldCopies = renderWorldCopies
+  }
+
+  override fun getRenderWorldCopies(): Boolean {
+    return map.renderWorldCopies
+  }
```

Comment:
Can you add tests for these too.

Reply:
@alexander-kulikovskii if you leave them here - they're not exposed to the final user. Please reach out to gl-native folks to understand whether we want to expose them or not and should they be experimental or not

Reply:
I'll expose it in separate PR